### PR TITLE
Add appveyor.yml

### PR DESCRIPTION
**Pull request by cadecairos**
This will enable us to run our CI on Windows.

Fixes https://github.com/MozillaFoundation/mofo-devops/issues/517

### CODE REVIEW

**Comment by cadecairos** [comment]
@gideonthomas I don't know why I put this review on you - GitHub probably suggested it. Should I ask someone else or are you able to take a look?

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+version: "{build}"
+
+branches:
+
```
Comment (gideonthomas) [inline appveyor.yml]: are these newlines needed?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - PYTHON: "C:\\Python36"
+      PYTHON_VERSION: "3.6.1"
+      PYTHON_ARCH: "32"
+
+init:
+  - "ECHO %PYTHON% %PYTHON_VERSION% %%PYTHON_ARCH"
+
+install:
+  - "virtualenv scienceapi -p %PYTHON%/python.exe"
+  - "./scienceapi/Scripts/activate"
```
Comment (gideonthomas) [inline appveyor.yml]: we're allowed to use UNIX paths?

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - PATH=C:\Program Files\PostgreSQL\9.6\bin\;%PATH%
+  - "pip install -r requirements.txt"
+
+services:
+  - postgresql
+
+before_test:
+  - "./scienceapi/Scripts/activate"
```
Comment (gideonthomas) [inline appveyor.yml]: why do we do this everytime?

**Comment by arush0311** [comment]
Works for me.

File: appveyor.yml (hunk 2 above)
Comment (cadecairos) [inline appveyor.yml]: Windows doesn't care about what path delimiter you use

File: appveyor.yml (hunk 1 above)
Comment (cadecairos) [inline appveyor.yml]: readability?

File: appveyor.yml (hunk 3 above)
Comment (cadecairos) [inline appveyor.yml]: I'm under the impression these scripts are executed separately from one another, so I'm not relying on some state existing between them.

**Comment by cadecairos** [comment]
@gideonthomas Anything you'd like me to change here?

**Pull request merged**